notes/benchmarks/benchmarks_kpis.md

```diff
@@ -0,0 +1,71 @@
+# Document and validate KPIs for Deku benchmarks
+
+Open [issue 574](https://github.com/marigold-dev/deku/issues/574)
+
+## Summary
+
+Define KPIs (Key Performance Indicators) to benchmark various parts of Deku (consensus, hashing...)
+
+## Motivation
+
+We want to measure Deku's ability to handle an important number of transactions. This number depends on several factors (number of validators, hardware, network...). We want to establish a protocol to make this measurements and find potential bottlenecks in the project.
+
+## Production
+
+We define the KPIs into 3 states, and by priority order.
+
+### State 1
+
+Focus on getting the number of TPS, hardware configurations, etc.
+ 
+There are two enviroments we will do the benchmark:
+- Local environment existing on a single machine
```

Comment:
For development only, right?

Reply:
Can you explain more? What do you mean by development only?

Reply:
What I mean by locally is what we are running the benchmarks on for instance, my laptop. And we will run that same tests on cloud.

Reply:
I was wondering if you were aiming for the same workloads on your laptop (with several instances of deku running!). It does not sound feasible, nor necessary, to me.

Reply:
Yes, not the same workloads of testing the hardware on my laptop.